model/map-hot-rod/src/main/java/org/keycloak/models/map/storage/hotRod/common/HotRodTypesUtils.java

```diff
@@ -110,4 +112,14 @@ public static String getKey(HotRodUserConsentEntity hotRodUserConsentEntity) {
     public static <T, V> Set<V> migrateSet(Set<T> p0, Function<T, V> migrator) {
         return p0 == null ? null : p0.stream().map(migrator).collect(Collectors.toSet());
     }
+
+    public static HotRodAuthenticationSessionEntity createHotRodAuthenticationSessionEntityFromMapEntry(Map.Entry<String, MapAuthenticationSessionEntity> entry) {
```

Comment:
This is unused after the change in MapAuthenticationSessionInterface (`Map` -> `Set`).